rpc/src/helpers.rs

```diff
@@ -106,3 +106,24 @@ impl FromStr for BlockHeaderInfo {
         })
     }
 }
+
+#[derive(Serialize, Debug, Clone)]
+/// Object containing information about the baking rights 
+pub struct BakingRights {
+    pub level: i32,
```

Comment:
Try to avoid pub fields if possible. We are using https://crates.io/crates/getset for easy field encapsulation.